src/main/windows/html_email.ts

```diff
@@ -292,7 +292,7 @@ img-src 'self' data:;
 media-src 'self' data:;
 style-src 'self' data: 'unsafe-inline';
 form-action 'none';
-script-src 'none';`
+script-src 'none';`.replace(/\n/g, '') // since new electron version there can not be linebreaks in the header anymore: https://github.com/deltachat/deltachat-desktop/issues/3701
```

Comment:
not sure about this specific language, however, in general, replacing new line by spaces seems to be better in case some language needs whitespace as a token. 

(assume eg html with <tag key=value\notherkey> )

Reply:
I think in this case the semicolon does this